Makefile

```diff
@@ -65,14 +65,20 @@ bin/cdi:
 	$(Q)echo "Building $@..."; \
 	cd cmd/cdi; $(GO_BUILD) -o $(abspath $@) .
 
+bin/validate:
+	$(Q)echo "Building $@..."; \
+	cd cmd/validate; $(GO_BUILD) -o $(abspath $@) .
+
 #
 # vendoring targets
 #
-vendor: ## ensure all the go.mod/go.sum files are up-to-date including vendor/ directory
+.PHONY: vendor
+vendor: ## ensure all the go.mod/go.sum files are up-to-date
```

Comment:
Let's just not call this 'vendor'. That strongly suggests that this would be doing a `go mod vendor`. Maybe 'mod-verify' ? Other than than it looks good to me.

Reply:
Sure. I copied the target and the name from the `containerd` Makfile. I believe it's called vendor there because they do vendor there. Maybe it makes sense to split this into `mod-tidy` and `mod-verify` targets.

Reply:
Updated.

Reply:
Since the mod-verify target also invokes the mod-tidy target, I'm resolving this.